vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml

```diff
@@ -6,21 +6,31 @@
         name: "*"
         state: latest
 
-    - name: Add OS release key
+    # TODO: (Sunnatillo) Remove this task after fully removing apt-key
+    - name: Remove OS old repository(without gpg key file location) 
```

Comment:
Nit: add space between `repository` and `(` for readability.

Reply:
done